data/map planets.txt

```diff
@@ -2268,6 +2271,7 @@ planet Iritoroost
 	attributes hai station uninhabited
 	landscape land/station23
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	Due to Iritoroost's relative isolation from any inhabited planets, the station has become something of a destination for those looking to do shady business deals or otherwise participate in unsavory activities without getting caught. As virtually all but the most basic systems are inactive, the lack of surveillance has proven attractive to those looking for an extra measure of privacy, whether that be for legal or illegal enterprises.`
```

Comment:
```suggestion
	description `	Despite Iritoroost's relative isolation from any inhabited planets, the station appears to be visited with remarkably frequency. It may have become something of a destination for those looking to do shady business, but the evidence left behind by visitors is eclectic. As virtually all but the most basic systems are inactive, it seems the lack of continuous surveillance has proven attractive to those looking for an extra measure of privacy, for whatever reasons they may have.`
```
A little too specific for something you should learn about from a person not a place.

Reply:
```suggestion
	description `	Despite Iritoroost's relative isolation from any inhabited planets, the station appears to be visited with remarkable frequency. It may have become something of a destination for those looking to do shady business, but the evidence left behind by visitors is eclectic. As virtually all but the most basic systems are inactive, it seems the lack of continuous surveillance has proven attractive to those looking for an extra measure of privacy, for whatever reasons they may have.`
```
Fixing a typo